Flutter/DateRangePicker/customizations.md

```diff
@@ -66,6 +66,9 @@ Widget build(BuildContext context) {
 
 ![Customizations Date Range Picker](images/customizations/customizations.png)
 
+>**NOTE**
+* Based on the given decoration, cell decoration will be rendered in the following order: blackout dates, special dates, disable dates, today date, weekend dates, trailing and leading dates, and dates decoration.
```

Comment:
Leading or trailing dates

Reply:
I have checked and updated in my latest commit